docs/site/migration/models/remoting-hooks.md

```diff
@@ -251,21 +291,238 @@ export class ProductController {
       },
     },
   })
-  async create(): // ...
-  Promise<Product> {
+  async create(/*...*/): Promise<Product> {
     // ...
   }
 }
 ```
 
 ## Accessing context data
 
-(to be done)
+In LoopBack 3, a remoting hooks receives a context object providing both
```

Comment:
This sentence is a bit long. 
Suggestion:
In LoopBack 3, a remoting hook receives a context object providing both transport-specific data, such as the HTTP request & response objects, and transport-independent data, such as the array of input arguments for the remote method and the result of remote method invocation.

Reply:
I am going to rework this paragraph into a list.

```markdown
In LoopBack 3, a remoting hook receives a context object providing:

- Transport-specific data like the HTTP request & response objects.
- Transport-independent data, for example the array of input arguments for the
  remote method and the result of remote method invocation.
```